README.md

```diff
@@ -18,29 +18,9 @@ CI will check that the code builds and the tests pass. In addition, it will run
 ## Style
 We use clang-format to enforce style in the C++ files. You can format all of your code using `catkin_make --make-args clangformat`. It will automatically format all of your .cpp and .h files.
 
```

Comment:
You should mention what the ./build.sh host does. For the people using it in the future.
That it runs on mac as well with docker daemon running, will allow compiling robosub repository in docker thus allowing cross-compilation. Will not be able to execute the code. The main purpose is cross-compilation. In the future, you will be implementing syncing.
If you want you can mention the steps to run linter, ie. (pylint , clang-format).